qa/tasks/cephfs/test_volume_client.py

```diff
@@ -14,11 +14,11 @@ class TestVolumeClient(CephFSTestCase):
     # One for looking at the global filesystem, one for being
     # the VolumeClient, two for mounting the created shares
     CLIENTS_REQUIRED = 4
-    py_version = 'python'
+    py_version = 'python3'
 
     def setUp(self):
         CephFSTestCase.setUp(self)
-        self.py_version = self.ctx.config.get('overrides', {}).get('python', 'python')
+        self.py_version = self.ctx.config.get('overrides', self.py_version)
```

Comment:
This change didn't make sense to me. The second argument is the default. You're getting the value of `overrides["python3"]`?

Reply:
I've amended this line.

However, I am not sure what it does; I couldn't step into this method by using pdb. I mildly recall that while looking into this line last time I found that python's executable was mentioned somewhere in a YAML file.

Reply:
https://github.com/ceph/ceph/tree/99826fbe755c534ec6a8b917c6a9c45c3e08c1e5/qa/suites/fs/basic_functional/tasks/volume-client/py